packages/drivers/odsp-driver/src/odspError.ts

```diff
@@ -33,6 +33,7 @@ export function errorObjectFromSocketError(
 		);
 
 		error.addTelemetryProperties({ odspError: true, relayServiceError: true });
+		error.endpointReachable = true;
```

Comment:
can you please move these two lines into createOdspNetworkError()?
I do not remember why, but I'd use same addTelemetryProperties() vs. setting a property directly - I think it should work correctly for this case (but please validate)

Reply:
I believe it's a right change, as there are two main calls to createOdspNetworkError(), another one being socket error, but I believe it happens only when got error on already established socket.

That said, it probably does not matter for sockets, as if we keep hitting issues with socket, they only matter when we can't connect to it (that's where back-off max matters), and thus it does not matter if we set endpointReachable for such case. But maybe (in future) some other workflows will use endpointReachable for something else - if it's save, I'd make sure it applies to socket flow as well.

Reply:
I have moved this setting in the createOdspNetworkError and set it to unconditionally true. However setting "relayServiceError" to true should be set here only as in createOdspNetworkError we don't know whether it is socket error or not.